tools/parity_scale_decoder/samples/trigger.json

```diff
@@ -5,15 +5,9 @@
             "Instructions": [
                 {
                     "Mint": {
-                        "object": {
-                            "Numeric": {
-                                "U32": 1
-                            }
-                        },
+                        "U32": 1,
                         "destination_id": {
-                            "Id": {
-                                "AssetId": "rose##alice@wonderland"
-                            }
```

Comment:
Given that it's unambiguously an AssetId, we could simplify the syntax further to just equal the Id.

Reply:
`Mint` isi is also available for `PublicKey` and `SignatureCheckCondition` so i don't think we can simplify further.